DFC.App.ContactUs.Services.EmailService/Contracts/IEmailRequestModel.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Collections.Generic;
+
+namespace DFC.App.ContactUs.Services.EmailService.Contracts
+{
+    public interface IEmailRequestModel
+    {
+        public string? GivenName { get; set; }
```

Comment:
Still don't agree with string?.  As string is a class it can be null hence no need for the question mark.